deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: infinispan-operator
       containers:
         - name: infinispan-operator
-          image: jboss/infinispan-operator:latest
+          image: docker.io/rigazilla/rigazilla-operator:i0.03
```

Comment:
Is this change intentional?

Reply:
Uh no, my fault... actually is the second time I made this mistake, I begin to think that I want to release a rigazilla-operator :)

Reply:
Why are you making the change in the first place?

Reply:
no just a mistake, the fix is in the commit below